examples/provisioner/bottlerocket.yaml

```diff
@@ -19,3 +19,15 @@ spec:
     karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
   securityGroupSelector:
     karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
+  blockDeviceMappings:
+    - deviceName: /dev/xvda
+      ebs:
+        volumeType: gp3
+        volumeSize: 2Gi
```

Comment:
one question for the data disk that seems to be undocumented too: What is the minimum size? The default is 5Gb according to what I see in EC2, you use 2, and usually we see almost zero usage 🤔

Reply:
@blackliner  i think the AMI is 20GiBs for the the data volume.. but i dont have my test cluster up to describe it at the moment
the root volume works as a read-only volume, that happens to allow in-place upgrades (if you had pets and not cattle)

Reply:
went in and looked
![image](https://user-images.githubusercontent.com/168366/184216493-2c078b0c-8152-4a34-b8b6-e412539d563d.png)

Reply:
The default in the AMI block device mappings for Bottlerocket is 2Gi for the regular AMI and 4Gi for the GPU AMI:

```
aws ec2 describe-images --image-ids ami-0af9a87b38254ecc7 --region us-east-2
{
    "Images": [
        {
            "Architecture": "x86_64",
            "CreationDate": "2022-07-28T21:04:29.000Z",
            "ImageId": "ami-0af9a87b38254ecc7",
            "ImageLocation": "amazon/bottlerocket-aws-k8s-1.22-x86_64-v1.9.0-159e4ced",
            "ImageType": "machine",
            "Public": true,
            "OwnerId": "419346874475",
            "PlatformDetails": "Linux/UNIX",
            "UsageOperation": "RunInstances",
            "State": "available",
            "BlockDeviceMappings": [
                {
                    "DeviceName": "/dev/xvda",
                    "Ebs": {
                        "DeleteOnTermination": true,
                        "SnapshotId": "snap-0717fba92f1252e9f",
                        "VolumeSize": 2,
                        "VolumeType": "gp2",
                        "Encrypted": false
                    }
                },
                {
                    "DeviceName": "/dev/xvdb",
                    "Ebs": {
                        "DeleteOnTermination": true,
                        "SnapshotId": "snap-030e1793d5439defe",
                        "VolumeSize": 20,
                        "VolumeType": "gp2",
                        "Encrypted": false
                    }
                }
            ],
            "Description": "bottlerocket-aws-k8s-1.22-x86_64-v1.9.0-159e4ced",
            "EnaSupport": true,
            "Hypervisor": "xen",
            "ImageOwnerAlias": "amazon",
            "Name": "bottlerocket-aws-k8s-1.22-x86_64-v1.9.0-159e4ced",
            "RootDeviceName": "/dev/xvda",
            "RootDeviceType": "ebs",
            "SriovNetSupport": "simple",
            "VirtualizationType": "hvm",
            "DeprecationTime": "2024-07-28T21:04:29.000Z"
        }
    ]
}
```